lib/experimental/inject_embed.dart

```diff
@@ -10,21 +10,44 @@ import 'package:logging/logging.dart';
 
 Logger _logger = Logger('dartpad-embed');
 
-// Use this location for local development:
-// var iframeSrc = 'embed-new.html?fw=true';
-var iframeSrc =
-    'https://dartpad.dartlang.org/experimental/embed-new.html?fw=true';
+// Use this prefix for local development:
+//var iframePrefix = '';
+var iframePrefix = 'https://dartpad.dartlang.org/experimental/';
```

Comment:
I left this as a variable so that the unit test can modify it before calling main(). I'm wondering if there is a better way to this. Maybe we should make it a class instead...